pwndbg/commands/hex2ptr.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+
+import argparse
+
+import pwndbg.color.message as M
+import pwndbg.commands
+from pwndbg.commands import CommandCategory
+from pwndbg.lib.common import hex2ptr_common
+
+# Define an argument parser for the command
+parser = argparse.ArgumentParser(
+    description="Converts a space-separated hex string to a little-endian address.",
+)
+parser.add_argument(
+    "hex_string", type=str, help="Hexadecimal string to convert (e.g., '00 70 75 c1 cd ef 59 00')."
+)
+
+
+@pwndbg.commands.ArgparsedCommand(parser, category=CommandCategory.MISC)
+def hex2ptr(hex_string) -> None:
+    combined_args = hex_string.replace(" ", "")
```

Comment:
just thinking why we have here replace, but not in:
```
@GdbFunction(only_when_running=True)
def hex2ptr(hex_string: gdb.Value | str) -> int:
```
?

Reply:
ah, I think I had it locally but accidentally deleted it when I was debugging gdb.Value and cleaning up all prints
Do I have to file new issue or I can PR straight away?

Reply:
yes, new pr